gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/FlowSpec.java

```diff
@@ -125,6 +125,18 @@ public static FlowSpec.Builder builder(URI catalogURI, Properties flowProps) {
       throw new RuntimeException("Unable to create a FlowSpec URI: " + e, e);
     }
   }
+
+  /**
```

Comment:
I prefer if we name this function something more semantic, such as `addMutableProperty` and a reader for it as `getMutableProperty` so that the user will not attempt to read from this.confgs.